ledger/participant-integration-api/src/main/scala/platform/store/backend/postgresql/PostgresEventStorageBackend.scala

```diff
@@ -25,18 +25,6 @@ class PostgresEventStorageBackend(ledgerEndCache: LedgerEndCache, stringInternin
         ParameterStorageBackendTemplate.participantAllDivulgedContractsPrunedUpToInclusive,
     ) {
 
-  // TODO FIXME: Use tables directly instead of the participant_events view.
-  override def maxEventSequentialIdOfAnObservableEvent(
-      offset: Offset
-  )(connection: Connection): Option[Long] = {
-    import com.daml.platform.store.Conversions.OffsetToStatement
-    // This query could be: "select max(event_sequential_id) from participant_events where event_offset <= ${range.endInclusive}"
-    // however tests using PostgreSQL 12 with tens of millions of events have shown that the index
-    // on `event_offset` is not used unless we _hint_ at it by specifying `order by event_offset`
-    SQL"select max(event_sequential_id) from participant_events where event_offset <= $offset group by event_offset order by event_offset desc limit 1"
```

Comment:
Out of interest, have you checked that in the new implementation that is no longer using `participant_events` view the index mentioned in the comment actually gets used?
(I'm guessing that's implied as benchmarks's number look good)